templates/default/cfnconfig.erb

```diff
@@ -15,6 +15,7 @@ cfn_node_type=<%= node['cfncluster']['cfn_node_type'] %>
 cfn_cluster_user=<%= node['cfncluster']['cfn_cluster_user'] %>
 <% if node['cfncluster']['cfn_node_type'] == 'ComputeFleet' -%>
 cfn_sqs_queue=<%= node['cfncluster']['cfn_sqs_queue'] %>
+cfn_health_sqs_queue=<%= node['cfncluster']['cfn_health_sqs_queue'] %>
```

Comment:
is this needed? I think the other queue is here to send the compute_ready event

Reply:
I thought this file need to be consistent with `attributes/default.rb` base on [this comment](https://github.com/aws/aws-parallelcluster-cookbook/blob/v2.6.1/attributes/default.rb#L266)

Reply:
I don't think it's needed but I'm ok with leaving it there